example.env.config.js

```diff
@@ -0,0 +1,48 @@
+/*
+Learner Record is now able to handle JS-based configuration!
+
+For the time being, the `.env.*` files are still made available when cloning down this repo or pulling from
+the master branch. To switch to using `env.config.js`:
+
+1. Make a copy of `example.env.config.js` and configure as needed.
+2. To ensure the port number is assigned to the one provided in your env.config,
+uncomment the lines in webpack.dev.config.js . This will eventually be resolved in https://github.com/openedx/frontend-build/issues/513
+
+Note: having both .env and env.config.js files will follow a predictable order, in which non-empty values in the
+JS-based config will overwrite the .env environment variables.
+
+frontend-platform's getConfig loads configuration in the following sequence:
+- .env file config
+- optional handlers (commonly used to merge MFE-specific config in via additional process.env variables)
+- env.config.js file config
+- runtime config
+*/
+
+module.exports = {
+  NODE_ENV: 'development',
+  PORT: 1990,
+  ACCESS_TOKEN_COOKIE_NAME: 'edx-jwt-cookie-header-payload',
+  BASE_URL: 'http://localhost:1990',
+  CREDENTIALS_BASE_URL: 'http://localhost:18150',
+  CSRF_TOKEN_API_PATH: '/csrf/api/v1/token',
+  ECOMMERCE_BASE_URL: 'http://localhost:18130',
+  LANGUAGE_PREFERENCE_COOKIE_NAME: 'openedx-language-preference',
+  LMS_BASE_URL: 'http://localhost:18000',
+  LOGIN_URL: 'http://localhost:18000/login',
+  LOGOUT_URL: 'http://localhost:18000/logout',
+  LOGO_URL: 'https://edx-cdn.org/v3/default/logo.svg',
+  LOGO_TRADEMARK_URL: 'https://edx-cdn.org/v3/default/logo-trademark.svg',
+  LOGO_WHITE_URL: 'https://edx-cdn.org/v3/default/logo-white.svg',
+  FAVICON_URL: 'https://edx-cdn.org/v3/default/favicon.ico',
```

Comment:
is it appropriate for these to be hardcoded domains instead of localhost domains? I know it's just an example but I feel like the examples should  really be examples (e.g. only use `example.org` and `localhost`). Same for line 43 below.

Reply:
[inform] This is the current convention we generally have in pretty every micro-frontend. The URLs used here are intended to be generic defaults (not edX specific, even though they're coming from an edX CDN url). The concern with using `localhost` domains here is that these images wouldn't exist and would display as missing images in the MFE, rather than generic defaults.

On stage/prod, we configure them to be the appropriate environment-specific URL, e.g. https://edx-cdn.org/v3/prod/logo.svg.

Reply:
it's a good point, and not one that I considered while migrating the environment variables from `.env.development`. The URLs come from the frontend-template [automatically provides them](https://github.com/openedx/frontend-template-application/blob/master/.env.development) when creating an MFE for the first time, so unless a team chose to change these values, all MFEs would start with the same URLs.